extensions/default/src/Toolbar/ToolbarLayoutSelector.tsx

```diff
@@ -61,12 +55,30 @@ function LayoutSelector({ rows, columns, servicesManager }) {
       label="Grid Layout"
       icon="tool-layout"
       onInteraction={onInteractionHandler}
+      className={className}
       dropdownContent={
         DropdownContent !== null && (
           <DropdownContent
             rows={rows}
             columns={columns}
             onSelection={({ numRows, numCols }) => {
+              if (
```

Comment:
I think my prettier formatter plugin put the if condition on a new line. Thoughts?

Reply:
I would suggest creating the callback function up above, and then just assigning it here.  That way you create it in a context that has more space and shouldn't break the spacing here.  IMHO, prettier produces extremely ugly code at times.

Reply:
Great suggestion. Thanks. I have just pushed in a change for this.